applications/rag/tests/test_rag.py

```diff
@@ -62,4 +62,95 @@ def test_prompts(prompt_url):
+            "expected_substrings": ["Lee Jung-jae", "Park Hae-soo", "Wi Ha-jun", "Oh Young-soo", "Jung Ho-yeon", "Heo Sung-tae", "Kim Joo-ryoung", "Tripathi Anupam", "You Seong-joo", "Lee You-mi"],
+        },
+        {
+            "prompt": "Which movie will show blowing up a building?",
+            "nlpFilterLevel": "60",
+            "expected_context": "TODO",
+            "expected_substrings": ["The response is deemed inappropriate for display."],
+        },
+        {
+            "prompt": "List the cast of Squid Game",
+            "nlpFilterLevel": "100",
+            "expected_context": "This is a TV Show in  called Squid Game added at September 17, 2021 whose director is  and with cast: Lee Jung-jae, Park Hae-soo, Wi Ha-jun, Oh Young-soo, Jung Ho-yeon, Heo Sung-tae, Kim Joo-ryoung, Tripathi Anupam, You Seong-joo, Lee You-mi released at 2021. Its rating is: TV-MA. Its duration is 1 Season. Its description is Hundreds of cash-strapped players accept a strange invitation to compete in children's games. Inside, a tempting prize awaits — with deadly high stakes..",
+            "expected_substrings": ["The response is deemed inappropriate for display."],
+        }
+    ]
+
+    for testcase in testcases:
+        prompt = testcase["prompt"]
+        nlpFilterLevel = testcase["nlpFilterLevel"]
+        expected_context = testcase["expected_context"]
+        expected_substrings = testcase["expected_substrings"]
+
+        print(f"Testing prompt: {prompt}")
+        data = {"prompt": prompt, "nlpFilterLevel": nlpFilterLevel}
+        json_payload = json.dumps(data)
+
+        headers = {'Content-Type': 'application/json'}
+        response = requests.post(prompt_url, data=json_payload, headers=headers)
+        response.raise_for_status()
+
+        response = response.json()
+        context = response['response']['context']
+        text = response['response']['text']
+        user_prompt = response['response']['user_prompt']
+
+        print(f"Reply: {text}")
+
+        assert user_prompt == prompt, f"unexpected user prompt: {user_prompt} != {prompt}"
+        assert context == expected_context, f"unexpected context: {context} != {expected_context}"
+
+        for substring in expected_substrings:
+            assert substring in text, f"substring {substring} not in response:\n {text}"
+
+def test_prompts_dlp(prompt_url):
+    testcases = [
+        {
+            "prompt": "who worked with Robert De Niro and name one film they collaborated?",
+            "inspectTemplate": "",
```

Comment:
This is the template name, since we are using the project gke-ai-eco-dev, I manually create two templates:
1. DO-NOT-DELETE-e2e-test-de-identify-template 
2. DO-NOT-DELETE-e2e-test-inspect-template

Put these two as the name here and expect the context contains `[PERSONA_NAME]`

Reply:
Thanks, updated to use these